scripts/pelorus-operator-patches/04_pelorus_cluster_service_versions.diff

```diff
@@ -106,18 +106,18 @@
 +
 +    ## Contributing to Pelorus
 +
-+    If you are interested in contributing to the Pelorus project, please review our Contribution guide which can be found in the [contribution guide](https://github.com/konveyor/pelorus/blob/master/CONTRIBUTING.md)
++    If you are interested in contributing to the Pelorus project, please review our Contribution guide which can be found in the [contribution guide](https://github.com/dora-metrics/pelorus/blob/master/CONTRIBUTING.md)
 +
 +    ## Statement of Support
 +
-+    Our support policy can be found in the [Upstream Support statement](https://github.com/konveyor/pelorus/blob/master/docs/UpstreamSupport.md)
++    Our support policy can be found in the [Upstream Support statement](https://github.com/dora-metrics/pelorus/blob/master/docs/UpstreamSupport.md)
 +
 +    ## Code of Conduct
-+    Refer to Konveyor's Code of Conduct [here](https://github.com/konveyor/community/blob/main/CODE_OF_CONDUCT.md).
++    Refer to dora-metrics's Code of Conduct [here](https://github.com/dora-metrics/community/blob/main/CODE_OF_CONDUCT.md).
```

Comment:
need to change?

Reply:
https://github.com/dora-metrics/pelorus/pull/910